pkg/expr/transform.go

```diff
@@ -11,11 +11,30 @@ import (
 	"github.com/grafana/grafana/pkg/components/simplejson"
 	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/tsdb"
+	"github.com/prometheus/client_golang/prometheus"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
+var (
+	expressionsQuerySummary *prometheus.SummaryVec
+)
+
+func init() {
+	expressionsQuerySummary = prometheus.NewSummaryVec(
+		prometheus.SummaryOpts{
+			Name:       "expressions_queries_duration_milliseconds",
+			Help:       "Expressions query summary",
+			Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
+		},
+		[]string{"respStatus"},
```

Comment:
Nit. Think this can be just status

```suggestion
		[]string{"status"},
```